### ISSUE TITLE
Vertical scrollbar appears when font is loaded after calendar

### ISSUE THREAD

**Issue by marko-rbn** [fullcalendar/fullcalendar#7491]
### Reduced Test Case

https://codepen.io/Marko-Ruban/pen/QWzPzEj

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

No connector (vanilla JS)

### Bug Description

When using font "ClearSans-Thin" (don't know about other fonts) a vertical scrollbar is visible after initialization (minimal configuration and no events).  Resizing the browser window (horizontally or vertically) recalculates the height attribute of table.fc-scrollgrid-sync-table element reducing it by 3px from original value and the scrollbar disappears.

I was unable to reproduce the issue directly in Codepen possibly because it does extra resizing of the iframe.  However, if you copy the URL of the iframe that shows the view and open it in a new tab, it will load with a scrollbar - see screenshot.  Sometimes it takes a page refresh to get the scrollbar to pop up.

https://cdpn.io/Marko-Ruban/fullpage/QWzPzEj

![screenshot](https://github.com/fullcalendar/fullcalendar/assets/16366658/1a2d3c36-002b-421b-bcf8-0e2b1ff5fde3)

**Comment by acerix** [comment]
Thanks for the test case, I can see the scrollbar in your codepen example.

I'm not sure it's actually a bug in the calendar though.   I believe you need to call `calendar.updateSize()` after the fonts are loaded to have the calendar update to the new size of the content, or wait until the fonts are loaded before initializing the calendar.

**Comment by marko-rbn** [comment]
Hello and thanks for quick response.  Your hint makes sense.  I tried preloading the font file using <link> tag, but that didn't change anything, I guess it still take a bit to load the font into CSS.  Then I tried waiting for document.fonts.status with this:

	waitForFonts();
	function waitForFonts() {
	  if (document.fonts.status == 'loading') setTimeout(waitForFonts, 50);
	  else calendar.render();
	}

That didn't work though, probably because it just waits for the file to load, as with the preload tag.
Finally, this old hack seems to work.

	setTimeout(function(){ calendar.updateSize(); }, 100);
